libs/virtual_joystick/joystick.kv

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) farm-ng, inc.
+#
+# Licensed under the Amiga Development Kit License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://github.com/farm-ng/amiga-dev-kit/blob/main/LICENSE
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+<VirtualJoystickWidget@Widget>:
```

Comment:
```suggestion
<VirtualJoystickWidget>:
```
I found that this raises a re-declaration warning in other apps because you declare in joystick.py too. Is it happening here too?

Reply:
Yes it does! Good find